modules/support-installing-packages-to-a-toolbox-container.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+[id="installing-packages-to-a-toolbox-container_{context}"]
+= Installing packages to a `toolbox` container
+
+By default, running the `toolbox` command starts a container with the `registry.redhat.io/rhel8/support-tools:latest` image.
+This image contains the most frequently used support tools.
+If you need to collect node-specific data that requires a support tool that is not part of the image, then you can install additional packages.
```

Comment:
ISG discourages using if-then in sentence structures. 

> ! if...then conj
Avoid using “then” to introduce an independent clause that follows an “if” clause. For example, write “If
you set a password, access is restricted,” not “If you set a password, then access is restricted.”

Reply:
Huge thank you.  I don't have all the ISG details memorized and wouldn't have even thought to look this one up.  Sincere thanks.